### ISSUE TITLE
Objective Sharpie pod init takes forever

### ISSUE THREAD

**Issue by Mikilll94** [xamarin/xamarin-macios#8238]
### Steps to Reproduce

1. Run `sharpie pod init ios AFNetworking` like in the docs here:
https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/objective-sharpie/examples/cocoapod?context=xamarin/ios
2. The installation hangs on this:

```
** Setting up CocoaPods master repo ...
   (this may take a while the first time)
Setup completed
** Searching for requested CocoaPods ...
```

### Expected Behavior
The installation should complete.

### Actual Behavior
The installation hangs.

### Environment
Objective Sharpie version: 3.4.0-c0f0e73
CocoaPods version: 1.9.1
macOS version: 10.15.3

**Comment by chamons** [comment]
Sharpie pod support is not in the best shape, I'd strongly suggest not using it and doing the sharpie work by hand.

FYI @dalexsoto

**Comment by Mikilll94** [comment]
@chamons 
Seriously? When this is going to be fixed? Manually doing bindings for complex Objective-C libraries is a disaster.

**Comment by dalexsoto** [comment]
@Mikilll94 I think there is a little misunderstanding here, what @chamons means is not doing the binding work by hand but doing the pod initialization yourself, we are deprecating `sharpie pod` tool but not `sharpie bind` unfortunately our cocoapods wrappers are too old and cocoapods has changed a lot and the way the tool is designed needs a lot of refactoring, it has been in our backlog for too long that we decided to invest this time in other areas that improve Xamarin.iOS/Mac experience. 

So TL;DR; you can still use objective sharpie to bind any Objective-C headers

**Comment by dogiaplinh** [comment]
@dalexsoto If sharpie pod was deprecated, what’s methods we could use to binding pod packages?

**Comment by chamons** [comment]
You would need to come up with your own sharpie command line invocation similar to the documented one [here](https://docs.microsoft.com/en-us/xamarin/ios/platform/binding-objective-c/walkthrough?tabs=macos#using-objective-sharpie).

Sometimes those require some knowledge of objective-c/the library in question, as you have to get the header path correct.

**Comment by alataffective** [comment]
I didn't have a problem when running `sharpie pod init ios AFNetworking` but had the hang problem described when running `sharpie pod init macosx CocoaFob`.

Regarding https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/objective-sharpie/get-started:

> NEW! version 3.0 also introduces a tool for easily binding CocoaPods via the sharpie pod command. If the library you're interested in is available as a CocoaPod, we recommend you start by attempting to bind the CocoaPod with Objective Sharpie (versus attempting to bind against the source directly).

If `sharpie pod` is being deprecated, is this still the right advice?

**Comment by nielscup** [comment]
@dalexsoto 
> we are deprecating `sharpie pod` tool but not `sharpie bind` unfortunately our cocoapods wrappers are too old and cocoapods has changed a lot and the way the tool is designed needs a lot of refactoring, it has been in our backlog for too long that we decided to invest this time in other areas that improve Xamarin.iOS/Mac experience.
> 
Is `sharpie pod` being deprecated or not? [This documentation](https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/objective-sharpie/examples/cocoapod?context=xamarin/ios) still recommends using `sharpie pod`

**Comment by LelicKAS** [comment]
Hey ... guys ...
I think it would be nice to clean up the documentation ... The documentation recommends a path that leads to a hang ....

**Comment by eman1986** [comment]
if pod is in fact not the best solution, you guys should update the doc to mention that tool is outdated, the way it's currently presented sounds like a new feature of sharpie and that we should use it.

I do feel the sharpie docs aren't as up to date as the other xamarin tools docs are, I understand you guys are busy but this sort of thing hurts people trying to work with your product and makes them think it's not production-grade work.
